news/2020-02-24-mappers-report-february.md

```diff
@@ -0,0 +1,134 @@
+
+With that being said, let's begin right away with the news!
+
+This month's issue of the Mappers' Report contains following topics:
+
+- **[Ranking Criteria Changes](#rcchanges)**
+- **[NAT Meetings](#natmeetings)**
+- **[Within the Beatmap Nominators and NAT](#withinbnnat)**
+- **[Mapping Contests](#mapping-contests)**
+- **[Summary](#summary)**
+
+## <a id="rcchanges"></a>Ranking Criteria Changes
+
+A new month brings a new avalanche of updates to the Ranking Criteria (RC). These past two months were especially productive with the implementation of rules!
+
+### General
+
+- **[Custom Difficulty Naming Updates](/wiki/Ranking_Criteria#rules.1):**
+A description has been added to the rule "A beatmapset's custom difficulty naming must follow a common theme or pattern related to the song…" to clarify its meaning.
+- **[Spread and Length Requierements](/wiki/Ranking_Criteria#rules.1):** The requirement that drain time must be 80% of play time has been removed from the point that "Difficulties lower than the highest difficulty can use their play time as a metric instead of drain time".
+- **[Sped Up Ver. Markers](/wiki/Ranking_Criteria#rules.2):** Songs that have been edited to have a higher tempo are now required to use a (Sped Up Ver.) marker in the title field. For songs where the genre is Techno, Trance, Dance, or other similar genres, (Nightcore Mix) will be used instead.
+- **[Hitsound File Length](/wiki/Ranking_Criteria#rules.4):** The minimum length for hitsound files has been reduced from 100ms to 25ms, after testing showed that 25ms hitsound files will behave correctly in-game.
+- **[Cutting Songs Guideline Changes](/wiki/Ranking_Criteria#guidelines.4):** Cuts are required to maintain the general impression and intensity of the full song, however the guideline about the cuts having to be 1:30 or above in length has been removed.
+- **[Background Requirements](/wiki/Ranking_Criteria#rules.5):** New minimum width and height sizes have been added in addition to the already present maximum width and height rules. The new minimum sizes are 160 pixels wide and 120 pixels high.
+- **[Background Quality Guideline](/wiki/Ranking_Criteria#guidelines.5):** A new guideline has been added specifying that background images must be of reasonable quality.
+
+### osu!taiko
+
+- **Drum Sampleset**: A rule was removed which forbade the use of the drum sampleset. Previously, the game had no drum sample set support for osu!taiko, but this was fixed and samples added. Thus, the rule has been removed as the sampleset is now usable.
+
+### osu!catch
+
+- **[HP Difficulty Guidelines](/wiki/Ranking_Criteria/osu!catch#difficulty-specific):** The difficulty specific guidelines for HP settings have been adjusted for Cup to Overdose to be in line with community standards.
+
+### osu!mania
```

Comment:
be sure to add https://osu.ppy.sh/community/forums/topics/1025805?start=unread to this section now o/